tools/base/fpgainfo-c/bmcinfo.c

```diff
@@ -247,63 +62,44 @@ fpga_result bmc_filter(fpga_properties *filter, int argc, char *argv[])
 	return res;
 }
 
-fpga_result bmc_command(fpga_token *tokens, int num_tokens, int argc,
-			char *argv[])
-{
-	(void)tokens;
-	(void)num_tokens;
-	(void)argc;
-	(void)argv;
-
-	fpga_result res = FPGA_OK;
-	struct bmc_info info;
-
-	int i = 0;
-	for (i = 0; i < num_tokens; ++i) {
-		res = get_bmc_info(tokens[i], &info);
-		ON_FPGAINFO_ERR_GOTO(res, out, 0);
-		print_bmc_info(&info);
-	}
-out:
-	return res;
-}
-
 Values *bmc_build_values(sensor_reading *reading, sdr_header *header,
 			 sdr_key *key, sdr_body *body)
 {
 	Values *val = (Values *)calloc(1, sizeof(Values));
 
+	(void)header;
+
 	if (NULL == val)
 		return NULL;
 
-	memset(val, 0, sizeof(val));
+	memset(val, 0, sizeof(*val));
```

Comment:
Doesn't `calloc` already zero the memory? If so, this isn't necessary.

Reply:
Good catch.